### ISSUE TITLE
Sqlalchemy properties throws error when sorted

### ISSUE THREAD

**Issue by shubham-padia** [miLibris/flask-rest-jsonapi#44]
Step to reproduce:
When using a hybrid property as follows:
```python
@hybrid_property
    def email(self):
        return self._email

    @email.setter
    def email(self, email):
        self._email = email
```
 In raw SQL query, you get order by table.email (the email column doesn't exist in the actual table) instead of order by table._email
A possible fix for this is here: https://github.com/shubham-padia/flask-rest-jsonapi/commit/3c622710168c97ed56d6a827ec7e75c2fe0ee9ca
If this seems okay, I can make a PR for it :) .

**Comment by akira-dev** [comment]
Yes you're right. I have used your PR to fix the issue. And your code is better than mine. Fixed in d22d5959aeada3eea8627a83157f4f96dea9936b

**Comment by akira-dev** [comment]
New release of Flask-REST-JSONAPI is available on Pypi 0.12.6
